### ISSUE TITLE
Individual secrets property names is not showing

### ISSUE THREAD

**Issue by ubersl0th** [aptakube/aptakube#2]
### Current and expected behavior

Currently the secrets show a label of `props.key` instead of the expected name/label for the secret. (see screenshot)

### Additional context

![image](https://user-images.githubusercontent.com/6698759/189877896-e17b48df-0ae0-455a-a853-0142fa80f46b.png)

### Operating System

macOS Monterey

### Environment

```
clientVersion:
  buildDate: "2022-08-23T17:36:43Z"
  compiler: gc
  gitCommit: a866cbe2e5bbaa01cfd5e969aa3e033f3282a8a2
  gitTreeState: clean
  gitVersion: v1.25.0
  goVersion: go1.19
  major: "1"
  minor: "25"
  platform: darwin/amd64
kustomizeVersion: v4.5.7
serverVersion:
  buildDate: "2021-11-15T08:35:41Z"
  compiler: gc
  gitCommit: 802eff1fe87ad2dd737ebbe891f30500b88beb00
  gitTreeState: clean
  gitVersion: v1.21.2
  goVersion: go1.16.5
  major: "1"
  minor: "21"
  platform: linux/amd64
```

**Comment by goenning** [comment]
Thanks for the report. I have a fix for this coming out later this week!

**Comment by goenning** [comment]
Fixed on 0.5.0

<img width="1374" alt="image" src="https://user-images.githubusercontent.com/94755/190485098-cd7844b7-2f31-4a97-b90c-3dd96fe2bcdf.png">
